harpy/commands/convert.py

```
@@ -304,34 +164,77 @@ def format_bc(bc):
-                    print_error("BX/VX tags present", f"The BX:Z and VX:i tags are already present in {os.path.basename(sam)} and does not need to be standardized.")
-                if record.has_tag("BX"):
-                    bx = record.get_tag("BX")
-                    if "0" in bx.split("_") or re.search(r"(?:N|[ABCD]00)", bx):
-                        record.set_tag("VX", 0, "i")
-                    else:
-                        record.set_tag("VX", 1, "i")
-                    continue
+    logtext = f"Standardizing [dim][-> [magenta]{style.lower()}[/]][/]" if style else "Standardizing"
+
+    if style:
+        bc_out = open("conversion.bc", "w")
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)